islandora_solr.module

```diff
@@ -136,6 +136,10 @@ function islandora_solr_permission() {
       'title' => t('Administer Solr'),
       'description' => t('Administer settings for the Solr client.'),
     ),
+    'view solr debug' => array(
```

Comment:
@DiegoPino Sure, I can create a constant... I did it this way because that's what it looked like the other permission settings were ('administer islandora solr', 'search islandora solr') . Are these defined as constants elsewhere? Or should they be turned into constants too?

Reply:
oh god, you are right 🛠 . Again, we have some inconsistencies between modules.... Yep, this one has no PHP Constants for permissions. Dismiss that part of the request but please add "islandora" somewhere to match the other permissions. Thanks!